README.md

```diff
@@ -87,12 +87,53 @@ jsep.removeIdentifierChar('@');
 
 ### Plugins
 JSEP supports defining custom hooks for extending or modifying the expression parsing.
-All hooks are bound to the jsep instance and called with a single argument and return void.
-The 'this' context provide access to the internal parsing methods of jsep
+Plugins are registered by calling `jsep.plugins.register(the_plugin)`.
+
+#### JSEP-provided plugins:
+* `ternary`: Built-in by default, adds support for ternary `a ? b : c` expressions
+* `object`: Adds object expression support: `{ a: 1, b: { c }}`
+
+#### How to add plugins:
+```javascript
+import jsep from 'jsep';
+import object from 'jsep/plugins/object.js';
+jsep.plugins.register(object);
```

Comment:
Why is `register()` needed? Importing the plugin should automatically register it, unless there's a reason not to.